### PR TITLE
Enhance troubleshooting

### PR DESCRIPTION
**Pull request by jenting**
Signed-off-by: JenTing Hsiao <jenting.hsiao@suse.com>

### CODE REVIEW

File: adoc/admin-troubleshooting.adoc [hunk 1]
```diff
@@ -45,4 +45,43 @@ cluster node it was created on.
...
+[source,bash]
+----
+skuba node bootstrap --user sles --sudo --target <IP/FQDN> <NODE NAME> -v=1
+----
+
+== Network Planning And Reconfiguration
+
+It is not possible to reconfigure the used IP ranges of SUSE CaaS Platform once the deployment is complete. Therefore, carefully plan for required IP ranges and future scenarios.
+
+Additionally, keep the network requirements in mind, as stated in Deployment Guide Networking chapter.
```
Comment (jenting) [inline adoc/admin-troubleshooting.adoc]: @nkoranova I don't know how to link the Deployment Guide Networking chapter.

**Comment by innobead** [comment]
@nkoranova please help review and merge. Also help @jenting question, thanks :+1:

File: adoc/admin-troubleshooting.adoc (hunk 1 above)
Comment (nkoranova) [reply adoc/admin-troubleshooting.adoc]: > @nkoranova I don't know how to link the Deployment Guide Networking chapter.

At the moment there is no great way to do that. We can't link between guides. I just write this: `refer to the _{productname} Deployment Guide, Networking chapter_.`.

We have a documentation tool chain feature on the way, that will make this possible though.

**Comment by nkoranova** [comment]
Ok, I've added small formatting edits. I am however not convinced that this is information for the Troubleshooting section. People go to the Troubleshooting section when they are already have the problem and are looking for a way to solve it. Telling them that they should have read the documentation better and that they should try harder next time is IMHO not very helpful and might just make them a bit angry. :D :innocent:  Would you be OK with me adding this as a warning or important notice to the Deployment Guide, Networking chapter instead?

**Comment by jenting** [comment]
> Ok, I've added small formatting edits. I am however not convinced that this is information for the Troubleshooting section. People go to the Troubleshooting section when they are already have the problem and are looking for a way to solve it. Telling them that they should have read the documentation better and that they should try harder next time is IMHO not very helpful and might just make them a bit angry. :D  Would you be OK with me adding this as a warning or important notice to the Deployment Guide, Networking chapter instead?

Agree!

**Pull request merged**